src/preprocessor/src/Preprocessor.cpp

```diff
@@ -47,7 +47,8 @@ std::unique_ptr<std::vector<std::string>> Preprocessor::SourceFileLoader::load(
         std::make_unique<std::vector<std::string>>();
     if (!fileStream.is_open()) {
         std::ostringstream ss;
-        ss << "Could not find source file defined at [" << file << " ]";
+        ss << "Could not find source file defined at [" << file << " ] in "
+           << std::experimental::filesystem::current_path();
```

Comment:
is using an experimental feature sensible?

Reply:
It's not meaningfully experimental. As it SHOULD be in C++ 14 upwards, yet somehow the only one who has managed to get it working without declaring as experimental is Alfie, the CI hasn't liked it either